wiki/en/Installation-for-Linux.md

```diff
@@ -13,26 +13,24 @@ Make sure you read the [Getting Started](Getting-Started) page.
 
 {% include infobox_each_os.html %}
 
-Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.
+**Upgrading?** You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.
 
-### Debian and Ubuntu
-
-We provide three equivalent `.deb` files for the most common architectures. Please download the appropriate one:
+**Installing a server?** Read the [guide for server adminstrators](Running-a-Server) first. 
 
-**For Intel/AMD based machines (amd64):**
+### Debian and Ubuntu
 
 [Download Jamulus (.deb, amd64)]({{ site.download_root_link }}{{ site.download_file_names.deb-gui }}){:.button}
 
-**For ARM based machines (e.g. Raspberry Pi, armhf, arm64):**
+**ARM-based machines** (e.g. Raspberry Pi): [armf]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) or [arm64]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-arm64 }})
 
-[Download Jamulus 32 bit (.deb, armhf)]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}){:.button}
-[Download Jamulus 64 bit (.deb, arm64)]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-arm64 }}){:.button}
 
 After you downloaded the correct file:
 
 1. *Ubuntu only* - Enable the Ubuntu "universe" repository (you can use the [GUI-based approach](https://askubuntu.com/a/148645) or [CLI-based approach](https://askubuntu.com/a/227788)).
 1. Update apt by opening a console window (CTRL+ALT+T should work) and type: `sudo apt-get update`
-1. Go to where you downloaded the installer and either double-click on it, or use the command line: `sudo apt install ./{{ site.download_file_names.deb-gui }}` for armhf: `sudo apt install ./{{ site.download_file_names.deb-gui-armhf }}` for arm64: `sudo apt install ./{{ site.download_file_names.deb-gui-arm64 }}`
+1. Go to where you downloaded the installer and either double-click on it, or use the command line: 
+
+	`sudo apt install ./{{ site.download_file_names.deb-gui }}` (or for Raspberry Pi etc. as above)  
```

Comment:
Or for what? Is this saying the Pi file differs?

Reply:
Yes, they are for different architectures. BTW you really need to be looking at this stuff rendered locally because literally only the text is being shown here in GH so you're missing chunks of info.

Reply:
could you tell me again how to render it locally? i can clone it locally.

Reply:
I plan to try: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll

Reply:
Maybe as first setup, yes. You‘ll need to run the po4a create all targets (?) or update templates (?) scripts to generate the translation

Reply:
`_po4a-tools/po4a-create-all-targets.sh` will create the translations.